plugins/inputs/aliyuncms/aliyuncms.go

```diff
@@ -6,6 +6,9 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/jmespath/go-jmespath"
+	"github.com/samber/lo"
```

Comment:
Can you please drop these down into the next section. In general we order imports by 1) go std lib 2) external

Reply:
I have removed the import and replaced it with the Go standard functionality “slices”.